### ISSUE TITLE
Mobile applications

### ISSUE THREAD

**Issue by saniaxxx** [Manuel83/craftbeerpi3#207]
Hello, is there an ios or android app for craftbeerpi v3 ?

**Comment by germanherbik** [comment]
yes, only for ios.

**Comment by germanherbik** [comment]
No, but you can conect to the raspberry to internet and then open vnc viewer on your ios or android phone.

**Comment by HooTsBrooklyn** [comment]
You don't need VNC if you are on the same network.  Simply open a browser window on another device and use the IP address of the Pi followed by port 5000

E.g.
http://192.168.1.7:5000
